src/Cameras/babylon.deviceOrientationCamera.ts

```diff
@@ -1,22 +1,10 @@
 module BABYLON {
     // We're mainly based on the logic defined into the FreeCamera code
     export class DeviceOrientationCamera extends FreeCamera {
-        //-- Begin properties for backward compatibility for inputs

```

Comment:
The only thing it might cause is missing functions in older VR DO Cameras, but shouldn't really be a problem. The constructor is the same, everything else as well.